<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlaceShipsForm</class>
 <widget class="QWidget" name="PlaceShipsForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>860</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QLabel#TurnLabel {
	color: #008af8;
	font-size: 30px;
	height: 30px;
	font-weight: bold;
}
QWidget#PlaceShipsForm {
	background: white
}</string>
  </property>
  <widget class="QLabel" name="TurnLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>225</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>Place your ships</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="BattleField" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>280</width>
     <height>280</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: transparent</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>400</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">height: 35px;
background: #869cff;
border-radius: 15px;
font-size: 16px;
color: white;
font-weight: 600;
</string>
   </property>
   <property name="text">
    <string>Start game</string>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>70</y>
     <width>401</width>
     <height>171</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="styleSheet">
       <string notr="true">margin: 15px;
font-size: 18px;
color: rgb(59, 50, 124);
font-weight: 600;</string>
      </property>
      <property name="text">
       <string>Ships left</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTop|Qt::AlignTrailing</set>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="ShipsStockForm" name="widget_2" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="DropArea" name="widget_3" native="true">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>280</width>
     <height>280</height>
    </rect>
   </property>
   <property name="acceptDrops">
    <bool>true</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background: yellow;</string>
   </property>
  </widget>
  <zorder>widget</zorder>
  <zorder>TurnLabel</zorder>
  <zorder>pushButton</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>widget_3</zorder>
 </widget>
 <customwidgets>
  <customwidget>
   <class>BattleField</class>
   <extends>QWidget</extends>
   <header>battlefield.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ShipsStockForm</class>
   <extends>QWidget</extends>
   <header>shipsstockform.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>DropArea</class>
   <extends>QWidget</extends>
   <header>droparea.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
